js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/AutoMaterializeExperimentalBanner.tsx

```diff
@@ -0,0 +1,51 @@
+import {Alert, Box, Colors, Icon, Popover, Tag} from '@dagster-io/ui-components';
+import React from 'react';
+import styled from 'styled-components';
+
+const AutoMaterializeExperimentalText = (
+  <span>
+    You can learn more about this new feature and provide feedback{' '}
+    <a href="https://docs.dagster.io/concepts/assets/asset-auto-execution#auto-materializing-assets-">
+      here
+    </a>
+    .
+  </span>
+);
+
+export const AutoMaterializeExperimentalBanner = () => {
+  return (
+    <Alert
+      intent="info"
+      title="Auto-materialize policies are experimental"
+      icon={<Icon name="info" color={Colors.Blue700} />}
+      description={AutoMaterializeExperimentalText}
+    />
+  );
+};
+
+export const AutoMaterializeExperimentalTag = () => {
+  return (
+    <Popover
+      content={<Container>{AutoMaterializeExperimentalText}</Container>}
+      hoverOpenDelay={100}
+      hoverCloseDelay={100}
+      placement="top"
+      interactionKind="hover"
+    >
```

Comment:
Ahh I like the idea of making the tag clickable and not showing the link inside the tooltip. I think in our design system normally this would be a white modal, but that felt kinda odd also.

Reply:
Yeah this feels much better!

![image](https://github.com/dagster-io/dagster/assets/1037212/2a453b5d-593f-4e62-8c90-4cd95aeb24eb)